test/framework/src/main/java/org/elasticsearch/test/rest/RestTestLegacyFeatures.java

```diff
@@ -20,11 +20,21 @@
  * production code anymore.
  */
 public class RestTestLegacyFeatures implements FeatureSpecification {
+
+    public static final NodeFeature ML_STATE_RESET_FALLBACK_ON_DISABLED = new NodeFeature("ml.state_reset_fallback_on_disabled");
+
     public static final NodeFeature FEATURE_STATE_RESET_SUPPORTED = new NodeFeature("system_indices.feature_state_reset_supported");
     public static final NodeFeature SYSTEM_INDICES_REST_ACCESS_ENFORCED = new NodeFeature("system_indices.rest_access_enforced");
 
     @Override
     public Map<NodeFeature, Version> getHistoricalFeatures() {
-        return Map.of(FEATURE_STATE_RESET_SUPPORTED, Version.V_7_13_0, SYSTEM_INDICES_REST_ACCESS_ENFORCED, Version.V_8_0_0);
+        return Map.of(
+            FEATURE_STATE_RESET_SUPPORTED,
```

Comment:
It may be worth changing to `Map.ofEntries(entry(...), entry(...))` so spotless groups the key & value together on the same line for readability

Reply:
Yes, done in https://github.com/elastic/elasticsearch/pull/102483, will merge them and use Map.ofEntries